awx/ui_next/src/screens/Credential/shared/CredentialFormFields/BecomeMethodField.jsx

```diff
@@ -52,7 +52,7 @@ function BecomeMethodField({ fieldOptions, isRequired }) {
           helpers.setValue(option);
           setIsOpen(false);
         }}
-        isExpanded={isOpen}
+        isOpen={isOpen}
```

Comment:
Could you add uniqueness id like this suggestion to the list?
```suggestion
        isOpen={isOpen}
        id='privilege-escalation-methods'
```